### DISCUSSION TITLE
buddhism/2025/prajnaparamita/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#166] [General]
# buddhism/2025/prajnaparamita/

https://shanenull.com/buddhism/2025/prajnaparamita/

**Comment by shane0** [comment]
# books

## edward conze

- https://terebess.hu/english/short.pdf
- http://lirs.ru/lib/conze/The_Large_Sutra_On_Perfect_Wisdom,Conze,1975.pdf

**Comment by shane0** [comment]
- Faxian (Fa-Hien) (c. 337–422 CE): A famous Chinese monk who traveled to India via the Silk Road to collect Buddhist scriptures. Faxian successfully returned to China but documented the extreme hardships of the journey, including the loss of companions.
- Xuanzang (Hsüan-tsang) (602–664 CE): Another celebrated monk who traveled to India along the Silk Road. Although Xuanzang survived his journey and returned to China, he wrote about others who faced fatal hardships during such trips.
- Yijing (I-ching) (635–713 CE): Traveled to India and beyond for Buddhist studies. Like Xuanzang, Yijing's records discuss the perilous conditions of the Silk Road that led to the deaths of many monks, but his own journey was successful.
- Mogao Monks: While individual names are not always recorded, the Mogao Caves at Dunhuang, an important Silk Road site, were associated with monks who lived in isolation or traveled the perilous route. Many monks who attempted to traverse the vast deserts or mountains for spiritual reasons faced death due to starvation, thirst, or exposure.

**Comment by shane0** [comment]
# 700 lines

- https://youtube.com/playlist?list=PL0w7f776wBnGkhiQILxCACok9Nx4xhvFA&si=nfoa0wsVkEWY76hF

**Comment by shane0** [comment]
# summaries

- **One word:** Emptiness  
- **Five words:** Wisdom transcending concepts and dualities  
- **Ten words:** Ultimate wisdom recognizing all phenomena as empty and interdependent
